src/ui/map.ts

```diff
@@ -121,15 +138,59 @@ class MapUI {
         }
       }
     };
+
+    const onStyleLoad = async () => {
+      MapUI.customMarkerRawSvg = undefined;
+      const style = map.getStyle();
+
+      const customMarkers = (style.metadata as any)?.['radar:customMarkers'];
+      if (Array.isArray(customMarkers) && customMarkers.length > 0) {
+        const customMarker = customMarkers[0]; // only support one custom marker for now
+        try {
+          const markerRawSvg = await Http.request({
```

Comment:
Rather than fetching the marker content, I wonder if we can just create an image element. Maybe there's some drawbacks to that.
```html
<img src="https://api.radar.io/maps/markers/${marker.id}?publishableKey=${publishableKey}" />
```